scipy/stats/_relative_risk.py

```diff
@@ -0,0 +1,260 @@
+
+import operator
+from collections import namedtuple
+import numpy as np
+from scipy.special import ndtri
+
+
+def _validate_int(n, bound, name):
+    msg = f'{name} must be an integer not less than {bound}, but got {n!r}'
+    try:
+        n = operator.index(n)
```

Comment:
```suggestion
        n = index(n)
```
if you agree w/limiting the import I'm suggesting above.